build.gradle

```diff
@@ -340,7 +340,7 @@ task replacecallTest(type: Test, dependsOn: 'copyJars') {
   workingDir = sourceSets.replacecallTest.output.resourcesDir
   testClassesDirs = sourceSets.replacecallTest.output.classesDirs
   classpath = sourceSets.replacecallTest.runtimeClasspath + files("$buildDir/libs/replacecall-${version}.jar")
-  jvmArgs "-javaagent:$buildDir/libs/replacecall-${version}.jar=--dont-transform=load-exclusions.txt,--debug"
+  jvmArgs "-javaagent:$buildDir/libs/replacecall-${version}.jar=--dont-transform=systemtest-load-exclusions.txt,--debug"
```

Comment:
This was the wrong file, which is why the test failed.

I've renamed the files and added a comment in the build script indicating where the file is located.